.github/workflows/ci.yml

```diff
@@ -548,6 +548,7 @@ jobs:
     with:
       project-name: OpenTelemetry.SemanticConventions
       code-cov-name: SemanticConventions
+      pack: false # No packages produced
```

Comment:
It is strange, SemanticConvention should be packageable

Reply:
For some reason it's explicitly marked as not:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/3d86bd1f379239a6a265948a07cb178e8b92934b/src/OpenTelemetry.SemanticConventions/OpenTelemetry.SemanticConventions.csproj#L8

I figured if that's wrong, that's probably a different discussion/PR.

Reply:
Thank you. Lets fix it first: #2683